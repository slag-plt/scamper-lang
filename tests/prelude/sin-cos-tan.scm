(sin 13)
(sin 0.71)
(sin 111)
(sin 4)
(sin 6.1)
(sin -5)
(sin -0.69)
(sin 89)
(sin 0)
(cos 13)
(cos 0.71)
(cos 111)
(cos 4)
(cos 6.1)
(cos -5)
(cos -0.69)
(cos 89)
(cos 0)
(tan 13)
(tan 0.71)
(tan 111)
(tan 4)
(tan 6.1)
(tan -5)
(tan -0.69)
(tan 89)
(tan 0)