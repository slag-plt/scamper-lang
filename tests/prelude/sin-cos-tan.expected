0.4201670368266409
0.6518337710215366
-0.8645514486106083
-0.7568024953079282
-0.18216250427209588
0.9589242746631385
-0.6365371822219679
0.8600694058124532
0
0.9074467814501962
0.7583618759905082
-0.5025443191453852
-0.6536436208636119
0.9832684384425845
0.28366218546322625
0.7712460149971067
0.5101770449416689
1
0.4630211329364896
0.8595286652169407
1.7203486651303583
1.1578212823495777
-0.18526223068913525
3.380515006246586
-0.8253361052690248
1.6858253705060158
0
